docs/source/index.mdx

```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
+* [Queries](https://www.apollographql.com/docs/react/data/queries/) and [Mutations](https://www.apollographql.com/docs/react/data/mutations/) (Apollo Client)
+* The [Apollo CLI](https://www.apollographql.com/docs/devtools/cli/) and [VS Code extension](./devtools/editor-plugins/)
+* [Get started with Apollo Graph Manager](https://www.apollographql.com/docs/graph-manager/getting-started/)
+
+</WorkoutMenuItem>
+
+<WorkoutMenuItem
+  icon={Weightlift}
+  keyWord="Develop"
+  otherWords="your graph"
+>
+
+* Caching in [Apollo Server](https://www.apollographql.com/docs/apollo-server/performance/caching/) and [Apollo Client](https://www.apollographql.com/docs/react/caching/cache-configuration/)
+* [Using schema directives](https://www.apollographql.com/docs/apollo-server/schema/directives/) (Apollo Server)
+* [Extending Apollo Server with plugins](https://www.apollographql.com/docs/apollo-server/integrations/plugins/)
+* Mocking new features in [Apollo Server](https://www.apollographql.com/docs/apollo-server/testing/mocking/) and [Apollo Client](https://www.apollographql.com/docs/react/development-testing/client-schema-mocking/)
+* [The schema registry](https://www.apollographql.com/docs/graph-manager/schema-registry/)
+
+</WorkoutMenuItem>
+
+<WorkoutMenuItem
+  icon={Paragliding}
+  keyWord="Deploy"
+  otherWords="to production"
+>
+
+* Deploying Apollo Server to [Heroku](https://www.apollographql.com/docs/apollo-server/deployment/heroku/), [AWS Lambda](https://www.apollographql.com/docs/apollo-server/deployment/lambda/), [Zeit Now](https://www.apollographql.com/docs/apollo-server/deployment/now/), etc.
+* Authentication in [Apollo Server](https://www.apollographql.com/docs/apollo-server/security/authentication/) and [Apollo Client](https://www.apollographql.com/docs/react/networking/authentication/)
+* Introduction to [Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/) and [Managed federation](https://www.apollographql.com/docs/graph-manager/managed-federation/overview/)
+* [Improving performance](https://www.apollographql.com/docs/react/performance/performance/) (Apollo Client)
+* [Preventing breaking changes to your graph](https://www.apollographql.com/docs/graph-manager/schema-validation/)
+
+</WorkoutMenuItem>
+
+<WorkoutMenuItem
+  icon={BadmintonPlayer}
+  keyWord="Debug"
+  otherWords="your solution"
+>
+
+* Testing in [Apollo Server](https://www.apollographql.com/docs/apollo-server/testing/testing/) and [Apollo Client](https://www.apollographql.com/docs/react/development-testing/testing/)
+* Error handling in [Apollo Server](https://www.apollographql.com/docs/apollo-server/data/errors/) and [Apollo Client](https://www.apollographql.com/docs/react/data/error-handling/)
+* [Metrics and logging](https://www.apollographql.com/docs/apollo-server/monitoring/metrics/) (Apollo Server)
+* [Segmenting metrics by client](https://www.apollographql.com/docs/graph-manager/client-awareness/)
```

Comment:
This doesn't really seem like a "debug" step but instead part of "production," no?